client/src/util/stateManager/matrix.ts

```diff
@@ -22,6 +22,7 @@ import { Matrix } from "./net-encoding/matrix";
 import { Column } from "./net-encoding/column";
 
 const utf8Decoder = new TextDecoder("utf-8");
+const DEFAULT_NUM_BINS = 5000;
```

Comment:
ideally this value would be communicated via the server config so we can ensure it stays in sync with the server encoding

Reply:
Actually I see it's only used for encoding here, but why do we ever have to encode on the client side?

Reply:
We used to encode on the client side when we allowed users to annotate the data (in desktop). I think we are planning at some point in the future to add that back to hosted explorer.